app/src/main/java/org/jellyfin/mobile/player/deviceprofile/DeviceProfileBuilder.kt

```diff
@@ -155,13 +159,17 @@ class DeviceProfileBuilder {
             }
         }
 
+        var subtitleProfiles = if (appPreferences.exoPlayerDirectPlayAss)
+            getSubtitleProfiles(EXO_EMBEDDED_SUBTITLES_ASS, EXO_EXTERNAL_SUBTITLES_ASS)
+        else getSubtitleProfiles(EXO_EMBEDDED_SUBTITLES, EXO_EXTERNAL_SUBTITLES);
```

Comment:
Can be a `val` (it's immutable) and should probably be a when statement for better readability:
```suggestion
        val subtitleProfiles = when {
            appPreferences.exoPlayerDirectPlayAss -> getSubtitleProfiles(EXO_EMBEDDED_SUBTITLES_ASS, EXO_EXTERNAL_SUBTITLES_ASS)
            else -> getSubtitleProfiles(EXO_EMBEDDED_SUBTITLES, EXO_EXTERNAL_SUBTITLES)
        }
```

*Also I truly like when statements :sweat_smile:*